dockerfiles/Dockerfile.rocm

```diff
@@ -17,26 +17,38 @@ RUN update-locale LANG=en_US.UTF-8
 ENV LC_ALL C.UTF-8
 ENV LANG C.UTF-8
 
-# Install rocm
-RUN apt-get update && apt-get install -y gnupg2 --no-install-recommends curl && \
-  curl -sL http://repo.radeon.com/rocm/rocm.gpg.key | apt-key add - && \
-  sh -c 'echo deb [arch=amd64] http://repo.radeon.com/rocm/apt/4.5.2/ ubuntu main > /etc/apt/sources.list.d/rocm.list'
-
-RUN apt-get update &&\
-    apt-get install -y sudo git bash build-essential rocm-dev libpython3.6-dev python3-pip miopen-hip rocblas half aria2 libnuma-dev
-
-RUN aria2c -q -d /tmp -o cmake-3.20.0-linux-x86_64.tar.gz \
-https://github.com/Kitware/CMake/releases/download/v3.20.0/cmake-3.20.0-linux-x86_64.tar.gz &&\
-tar -zxf /tmp/cmake-3.20.0-linux-x86_64.tar.gz --strip=1 -C /usr
-
-# Install yapf
-RUN pip3 install yapf==0.28.0 numpy
-
-ENV PATH /opt/miniconda/bin:/code/cmake-3.20.0-linux-x86_64/bin:${PATH}
-
-# Install dependencies
-RUN apt-get update &&\
-    apt-get install -y rocrand rccl hipsparse hipfft hipcub hipblas rocthrust
+# Register the ROCm package repository
+ARG ROCM_VERSION=5.2.3
+ARG AMDGPU_VERSION=22.20.3
+
+# Install ROCm and rocm-dev package
+RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates curl libnuma-dev gnupg \
+  && curl -sL https://repo.radeon.com/rocm/rocm.gpg.key | apt-key add - \
+  && printf "deb [arch=amd64] https://repo.radeon.com/rocm/apt/$ROCM_VERSION/ ubuntu main" | tee /etc/apt/sources.list.d/rocm.list \
+  && printf "deb [arch=amd64] https://repo.radeon.com/amdgpu/$AMDGPU_VERSION/ubuntu focal main" | tee /etc/apt/sources.list.d/amdgpu.list \
+  && apt-get update \
+  && apt-get install -y --no-install-recommends \
+  sudo \
+  git \
+  libelf1 \
+  kmod \
+  file \
+  python3 \
```

Comment:
Why do you install `python3` here?  Does this docker file use miniconda or system installed python for building onnxruntime?

Reply:
Install miniconda in this dockerfile and install the build onnxruntime package.
Now use ROCm official docker images as base image instead of install all the packages manually.